editor/plugins/texture_editor_plugin.cpp

```diff
@@ -44,6 +44,15 @@ TextureRect *TexturePreview::get_texture_display() {
 	return texture_display;
 }
 
+void TexturePreview::set_texture_display_outline_width(int p_width) {
+	outline_width = p_width;
+
+	margin_container->add_theme_constant_override("margin_right", outline_width + 1);
+	margin_container->add_theme_constant_override("margin_top", outline_width + 1);
+	margin_container->add_theme_constant_override("margin_left", outline_width + 1);
+	margin_container->add_theme_constant_override("margin_bottom", outline_width + 1);
```

Comment:
I don't get it, what these `+ 1` are for? Does it look better this way or something like that? :thinking:

Reply:
Yeah, this is very subjective.
| With additional gap | Without gap |
| --- | --- |
| ![with_plus_1](https://github.com/user-attachments/assets/4f2a3259-d4b6-4a41-a64a-6d496f5028b5) | ![without_plus_1](https://github.com/user-attachments/assets/5df6dca5-2de4-4f64-9cf1-1543893572ed) |

So, maybe I'll just remove this hardcoded `+1`

Reply:
@kleonc I removed `+1`

Reply:
That's indeed subjective (briefly they're the same picture :smile:). I tend to prefer less margins so I'd opt for no `+1`, but if it would be left I wouldn't really care I think. :upside_down_face: (but in such case a comment in the code would be welcome to explaing why `+1` are there)